model/model/v1alpha/common.proto

```diff
@@ -2,66 +2,65 @@ syntax = "proto3";
 
 package model.model.v1alpha;
 
-// Google api
+// Google API
 import "google/api/field_behavior.proto";
 
-// BoundingBox represents the bounding box data structure
+// BoundingBox represents a frame in an image that can be drawn when detecting
+// patterns in objects.
 message BoundingBox {
-  // Bounding box top y-axis value
+  // Top Y-axis.
   float top = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
-  // Bounding box left x-axis value
+  // Left X-axis.
   float left = 2 [(google.api.field_behavior) = OUTPUT_ONLY];
-  // Bounding box width value
+  // Width.
   float width = 3 [(google.api.field_behavior) = OUTPUT_ONLY];
-  // Bounding box height value
+  // Height.
   float height = 4 [(google.api.field_behavior) = OUTPUT_ONLY];
 }
 
-// Additional hyperparameters for model inferences
```

Comment:
@heiruwu I deleted this definition as I couldn't see any reference to it within this repo or `model-backend`. Can you confirm this is unused? (Kind of makes sense as protobuf `Struct` is more suited for this structure).

Reply:
I believe this is added by @tonywang10101, let's asked for his input.

Reply:
Yes, this one should be removed. We now use a struct type for additional hyperparameters, so there's no need for an extra definition in common.proto

Reply:
🧹